docs: rework md book (#215)
quay/claircore: docs/reference/manifest.md

# Manifest

> A Manifest defines a container image, the order of the image's layers, and how to retrieve the layers.

Reviewer: ```suggestion
A Manifest is analogous to an [OCI Image Manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md): it defines the order of layers and how to retrieve the them.
```
